template/index.js

```diff
@@ -4,6 +4,5 @@
 
 import {AppRegistry} from 'react-native';
 import App from './App';
-import {name as appName} from './app.json';
 
-AppRegistry.registerComponent(appName, () => App);
+AppRegistry.registerComponent('HelloWorld', () => App);
```

Comment:
you could take `placeholderName` from `template.config.js`

Reply:
scratch that, forgot we aim for easy replacement of template name and don't want to tie template to the config anyhow

Reply:
Can you elaborate? I don't think we should hardcode `HelloWorld` anywhere.

Reply:
This is a placeholder, that will be globally replaced in the initialization process. This placeholder is used everywhere in the template: ios, android etc.

Reply:
Got it!